src/Spryker/Zed/EventBehavior/Business/Model/EventResourceRepositoryManager.php

```diff
@@ -183,18 +200,89 @@ protected function getIdColumnName($plugin): ?string
         return $idColumnName[1] ?? null;
     }
 
+    /**
+     * @param \Spryker\Zed\EventBehavior\Dependency\Plugin\EventResourcePluginInterface $plugin
+     * @param array<\Spryker\Shared\Kernel\Transfer\AbstractTransfer> $transfers
```

Comment:
Those are never "Just" the abstract one - at least those are usually invalid to be used for methods that do not exist
Are you only using methods from the abstract one here?